### PR TITLE
describe how we use the "next" branch to incrementally change workflows

### PR DESCRIPTION
**Pull request by marten-seemann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,4 +18,10 @@ In order to help with the distribution of these workflows, this repository defin
 ## Usage
 
 Workflows are distributed to all repositories listed in [config.json](config.json).
+
 If you want your project to participle, please send a PR!
+
+## Development
+
+The master branch contains currently deployed workflows.
+When we make minor changes to these workflows, we don't always want these changes to get deployed to all hundreds of repositories, as this creates a lot of unnecessary noise. Minor changes to the workflows are therefore merged into the [next](https://github.com/protocol/.github/tree/next) branch. When the time has come, we create a PR from the next branch to master to trigger a deployment to all repositores.
```
Comment (mvdan) [inline README.md:27]: nit: backquote the names for clarity, like:

> we create a PR from the `next` branch to `master` to...

**Review by mvdan — APPROVED** [review]

**Review by Stebalien — APPROVED** [review]
Future work: make it possible to designate some repos as "next" (for testing).

File: README.md (hunk 1 above)
Comment (marten-seemann) [reply README.md]: Fixed.

**Comment by marten-seemann** [comment]
> Future work: make it possible to designate some repos as "next" (for testing).

What does that do with those repos (if anything)?

**Pull request merged**